db/migrate/20160626235137_create_join_table_category_keyword.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCategoryKeyword < ActiveRecord::Migration
+  def change
+    create_join_table :Categories, :Keywords do |t|
```

Comment:
`ActiveRecord` appears to normalize them, but I think [the parameters are conventionally lowercase](http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/create_join_table) (as the table names are).